src/app/models/types/curationData.ts

```diff
@@ -34,8 +35,8 @@ export type VisualStyle = keyof typeof VISUAL_STYLE;
 export type VisualProminence = keyof typeof VISUAL_PROMINENCE;
 
 export interface CurationProps {
-  visualStyle: VisualStyle;
-  visualProminence: VisualProminence;
+  visualStyle: Partial<VisualStyle>;
+  visualProminence: Partial<VisualProminence>;
```

Comment:
Hmm I don't think this is the intended usage here - can we remove?
```suggestion
  visualStyle: VisualStyle;
  visualProminence: VisualProminence;
```